app/javascript/components/redux/message.js

```diff
@@ -0,0 +1,19 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const fetchRandomMessage = createAsyncThunk('/message/random', async () => {
+	const response = await fetch('http://localhost:3000/api/v1/random');
```

Comment:
- [ ] Kindly change the URL to `api/v1/random` otherwise ı can't see the random greeting on the page.

Reply:
hello @ahmetbozaci , to see the random message on the screen, simply visit the root route.

I have attached a loom video explaining quite well how to see all the messages from the end point and on the screen.

Here is the [link](https://www.loom.com/share/84fdf46b971b431f8ec8a4be90dee4cb) to the loom video.

Reply:
@billionsjoel  In the loom video ı can't see the URL. 
I can see the message ın this link `http://127.0.0.1:3000/` or [localhost](url) only if ı change the URL to `api/v1/random.` 

- [ ] You can think of it as a real project. Changing the URL to see the message is not good for the users.